volttrontesting/utils/platformwrapper.py

```diff
@@ -819,7 +822,10 @@ def install_agent(self, agent_wheel=None, agent_dir=None, config_file=None,
     def restore_conf(self):
         if self.rmq_conf_backup:
             _log.debug("Restoring original rabbitmq.conf to server and restarting rmq")
-            shutil.move(self.rmq_conf_backup, os.path.join(rabbitmq_config["rmq-home"],'etc/rabbitmq/rabbitmq.conf'))
+            try:
+                shutil.move(self.rmq_conf_backup, os.path.join(rabbitmq_config["rmq-home"],'etc/rabbitmq/rabbitmq.conf'))
+            except IOError as e:
```

Comment:
When does this happen - we have a valid not null self.rmq_conf_backup but the file doesn't exist?

Reply:
This behavior is seen many times specially after running certs related tests.

Reply:
Removing try-except to investigate actual problem